<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditCalculatorWindow</class>
 <widget class="QDialog" name="creditCalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kalkulator</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>338</width>
     <height>296</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="creditCalculatorLabel">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight SemiConde</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kalkulator kredytu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="startDateLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Data zawarcia umowy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="financialPeriodLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Okres finansowania</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="financialPeriodInput">
        <property name="text">
         <string>NNN miesięcy</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="investmentValueLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wartość inwestycji</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="investmentValueInput">
        <property name="text">
         <string>NNN NNN NNN, NN zł</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="percentageLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Oprocentowanie kredytu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="percentageInput">
        <property name="text">
         <string>NN, NN%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="commisionLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Prowizja</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commisionInput">
        <property name="text">
         <string>NN, NN%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Wyczyść</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateScheduleButton">
        <property name="text">
         <string>Generuj harmonogram</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gobackButton">
        <property name="text">
         <string>Wróć</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
